setup.py

```diff
@@ -1,7 +1,12 @@
+import os
+
 from setuptools import setup, find_packages
 
-__version__ = '1.0.16'
+__version__ = '1.0.17'
 
+requirements_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'requirements.txt')
+with open(requirements_path) as requirements_file:
+    requirements = requirements_file.readlines()
```

Comment:
actually there is some concern over pin the deps in https://github.com/lyft/amundsen/issues/80